corehq/apps/users/views/web.py

```diff
@@ -181,10 +181,11 @@ def __call__(self, request, uuid, **kwargs):
                         created_via=USER_CHANGE_VIA_INVITATION,
                         domain=invitation.domain,
                         is_domain_admin=False,
+                        commit=False
                     )
-                    user.save()
```

Comment:
I'm not seeing where the user gets saved now if `commit=False` is passed down the call stack?